includes/class-evf-form-task.php

```diff
@@ -103,8 +103,8 @@ public function do_task( $entry ) {
 			}
 
 			// reCAPTCHA check.
-			$site_key   = get_option( 'evf_recaptcha_site_key', '' );
-			$secret_key = get_option( 'evf_recaptcha_site_secret', '' );
+			$site_key   = get_option( 'everest_forms_recaptcha_site_key', '' );
```

Comment:
Why is empty default value set? Simply, get_option( 'everest_forms_recaptcha_site_key' ) should have checked the non-empty value.